docs/UserGuide.md

```diff
@@ -783,9 +783,9 @@ _________________________________________________________
 
 # <a id="csv"></a> &nbsp;&nbsp; CSV: `csv`
 
-If our statistical analysis feature is not good enough for you, you may convert the database records
-into a csv file by simple keying in the `csv` command. You will then be able to use Excel to do advanced
-statistical analysis.
+> 😃 If our statistical analysis feature is not good enough for you, you may convert the database records
+> into a csv file by simply keying in the `csv` command. You will then be able to use Excel to do advanced
+> statistical analysis.
```

Comment:
would be better if changed to: "This feature allows you to export your data into a CSV file, for users who desire more flexibility with customizing their data and further analysis which is not yet supported by Budget Tracker."

Reply:
Ok!